### DISCUSSION TITLE
Shape Template Cheatsheet - a quick way to lookup OC template matching expressions

### DISCUSSION THREAD

**Discussion by pbull001** [OrchardCMS/OrchardCore#17223] [Show and tell]
Hi Team,

I created this cheat sheet for myself a while a go. It helps looking and identifying the match expression you want to use with different shapes.

I don't guarantee it to be 100% correct or to date - high degree of correctness for v1.8.3.

Did look at adding to docs but the MD for the tables looked to painful :) 

[UnpackingOCTemplates.xlsx](https://github.com/user-attachments/files/18110628/UnpackingOCTemplates.xlsx)

**Comment by hishamco** [comment]
Thanks @pbull001 make it as PDF with some formatting will be more useful, this could be shared and pinned in Discord
